Example/TABTestKit_ExampleUITests/ExampleFeature.swift

```diff
@@ -22,7 +22,42 @@ class ExampleFeature: TestBase {
       Then(I: seeTheExampleScreen)
     }
   }
+}
+
+class BiometricTests: XCTestCase {
+  
+  private var app: XCUIApplication!
+  
+  private lazy var authenticateButton = app.buttons.element(matching: .button, identifier: "authenticateButton")
+  private lazy var authenticateLabel = app.staticTexts.element(matching: .any, identifier: "biometricStatusLabel")
+  
+  override func setUp() {
+    app = XCUIApplication()
+  }
+
+  override func tearDown() {
+    app = nil
+  }
   
+  func test_launchWithBiometricsDisabled() {
+    performUnenrollment()
+    app.launch()
+    Scenario("Launching the app with biometrics disabled") {
+      Given(I: seeBiometricsUnavailableStatus())
+      Given(I: seeAuthenticateButtonIsDisabled())
+    }
+  }
+  
+  func test_biometricsEnabled() {
+    performEnrollment()
+    app.launch()
+    Scenario("Enroll biometrics, background & foreground the app and then perform successful authentication") {
+      Given(I: seeBiometricsAvailableStatus)
+      Given(I: seeAuthenticateButtonIsEnabled)
```

Comment:
Can't have two `Given`s in the same scenario. 
Max one Given, When and Then, with multiple Ands if you need them

Reply:
There are also another initialiser for steps like `Given(the:`, so you can have a step read:

`And(the: authenticationButtonIsEnabled)`